packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts

```diff
@@ -349,6 +351,26 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
         );
     }
 
+    /**
+     * Insert the image editing wrapper in the selected image
+     */
+    public insertImageEditingWrapper() {
+        if (this.editor) {
+            const selection = this.editor.getDOMSelection();
+            if (!this.editor.getEnvironment().isSafari) {
```

Comment:
we should also only do focus if selection is image, right? so move this under line 363